### DISCUSSION TITLE
怎么修改打包后index.html的输出路径

### DISCUSSION THREAD

**Discussion by honorsuper** [umijs/umi#7342] [Q&A]
怎么修改打包后index.html的输出路径

**Comment by xlei1123** [comment]
output: path. 看webpack吧

**Reply by honorsuper** [reply]
output是修改整个打包后资源的目录，我这边是要单独修改html的路径。如果是使用过htmlWebpackPlugin的话，这边是可以修改插件里面的fileName，但umi这边没有对应的配置

**Comment by Yourrunner** [comment]
我这边有一个问题，可不可以在webpack的插件中拿到umi打包过程中动态生成的index.html文件呢？我尝试了很多方法，出了umiplugin提供的hook其他方法都拿不到这个文件，大家有没有什么好的办法呢？
